google/cloud/pubsub/samples/samples.cc

```diff
@@ -669,6 +675,171 @@ void ExampleStatusOr(google::cloud::pubsub::TopicAdminClient client,
   (std::move(client), argv.at(0));
 }
 
+void CreateAvroSchema(
+    google::cloud::pubsub_experimental::SchemaAdminClient client,
+    std::vector<std::string> const& argv) {
+  //! [START pubsub_create_avro_schema] [create-avro-schema]
+  namespace experimental = google::cloud::pubsub_experimental;
+  [](experimental::SchemaAdminClient client, std::string const& project_id,
+     std::string const& schema_id) {
+    auto constexpr kDefinition = R"js({
+      "type": "record",
+      "name": "State",
+      "namespace": "utilities",
+      "doc": "A list of states in the United States of America.",
+      "fields": [
+        {
+          "name": "name",
+          "type": "string",
+          "doc": "The common name of the state."
+        },
+        {
+          "name": "post_abbr",
+          "type": "string",
+          "doc": "The postal code abbreviation of the state."
+        }
+      ]
+    })js";
+    auto schema = client.CreateAvroSchema(
+        experimental::Schema(project_id, schema_id), kDefinition);
+    if (!schema) throw std::runtime_error(schema.status().message());
```

Comment:
Were these "create" calls cases where we should document tolerating "already exists"?

Reply:
yes, fixed. Also fixed the `DeleteSchema()` and `kNotFound` case.